components/StatusItem/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { Text, View, StyleSheet, Image } from 'react-native';
+import { Constants } from 'expo';
+import PropTypes from 'prop-types';
+
+
+class Icon extends Component {
+  render() {
+    if (this.props.danceStatus == 'Uploaded') {
+    pic=require('..\\icons\\checkmark.png')
```

Comment:
line 10 - line 13 is not indented enough

Reply:
seems like you're just setting a value on `pic` without first declaring it with `var`/`let`/`const` etc. which isn't best code practice.  pls do that first :) 

lmk if you have any questions!